dotcom-rendering/src/layouts/TagPageLayout.tsx

```diff
@@ -185,7 +185,9 @@ export const TagPageLayout = ({ tagPage, NAV }: Props) => {
 					image={tagPage.header.image}
 				/>
 				{tagPage.groupedTrails.map((groupedTrails, index) => {
-					const locale = getEditionFromId(tagPage.editionId).locale;
+					const locale = getEditionFromId(
+						tagPage.editionId,
+					).dateLocale;
```

Comment:
Looking at this, I wonder if destructuring might be both explicit and shorter.

```suggestion
					const { dateLocale } = getEditionFromId(
						tagPage.editionId,
					);
```